frontend/src/metabase/query_builder/components/QuestionActions.unit.spec.tsx

```diff
@@ -46,22 +52,42 @@ function setup({ card }: { card: Card }) {
   );
 }
 
-describe("Question Actions | Icons", () => {
-  ["structured", "native"].forEach(queryType => {
-    iconList.forEach(({ label, tooltipText }) => {
-      it(`should display the "${label}" icon with the "${tooltipText}" tooltip for ${queryType} questions`, async () => {
-        setup({
-          card:
-            queryType === "structured"
-              ? TEST_STRUCTURED_CARD
-              : TEST_NATIVE_CARD,
-        });
+describe("QuestionActions", () => {
+  it.each(ICON_CASES)(
```

Comment:
This it to make it webstorm-friendly - see the green arrow to run the test
<img width="916" alt="Screenshot 2023-07-10 at 17 21 55" src="https://github.com/metabase/metabase/assets/8542534/63c6141a-ce33-47f3-b65e-eb418c358464">

<img width="769" alt="Screenshot 2023-07-10 at 17 22 09" src="https://github.com/metabase/metabase/assets/8542534/a5f0817b-1299-4fcc-a279-3035a82de977">